### ISSUE TITLE
Grades Button Missing

### ISSUE THREAD

**Issue by Nebelung-Dev** [ksucpea/bettercanvas#33]
When using Better Canvas my `View Grades` button is missing.

**Comment by ksucpea** [comment]
That's weird, my view grades button still appears. Do you have the better to do list enabled?

**Comment by Nebelung-Dev** [comment]
Yeah I just tested it and `better to do list` is causing the problem
